src/js/jwplayer.js

```diff
@@ -1,4 +1,5 @@
 import 'polyfills/promise';
+import 'polyfills/focus';
```

Comment:
Please import this in controls. It's not needed outside of controls, and would exclude it from the headless player without special logic.